src/ATen/function_wrapper.py

```diff
@@ -222,11 +235,17 @@ def translate_formal(argument, option):
             'type': type_str,
             'dynamic_type': DYNAMIC_TYPE.get(argument['type'], argument['type']),
         }
+        if 'default' in argument:
+            if 'if_true' in argument:
```

Comment:
Similar thoughts here (though this might not be your fault) - what does `if_true` mean? I have no idea...

Reply:
In ATen these options are `bool`. In TH they're the `if_true` values.

i.e. `symeig` has a parameter `char upper` in TH which can be `U` or `L`. In ATen (and PyTorch) it's `bool` upper.